prisma/schema.prisma

```diff
@@ -0,0 +1,28 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+
+model ListItem {
+  id        String   @id @default(uuid())
+  name      String
+  description String?
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt  
+  authorId  String
+
+  @@index([authorId])
```

Comment:
Since Clerk is being used to manage our users, we don't need a User table in our database. However, we still want to query items based on their relationship to the user, so we should add an index on the relationship keys to make sure these queries remain performant.